core/test/regression/api/canary/admin/utils.js

```diff
@@ -37,8 +37,6 @@ const expectedProperties = {
         .without('visibility')
         .without('password')
         .without('locale')
-        .without('ghost_auth_access_token')
```

Comment:
Think it makes sense to do the same thing in other utils as well. My thinking was that these `without` are guarding from exposing fields that we know are in the DB but should never make it outside the API. Now that this fields is dropped completely there's no reason to keep it anywhere (for example this without is still in acceptance/old/admin/utils.js etc.). Wdyt?

Reply:
@gargol Yeah I was thinking about this and had removed them initially, but was not sure if older tests should be kept up-to-date with DB changes for 3.0, specially in the case where not changing them is not breaking anything. My other concern was setting the precedent for future cleanups and so left it as-is for second opinion. 🤔 I have no strong pref in this case though, so will update the older tests if you think that works better :)

Reply:
hm. you have a point about not touching old version tests in case they don't get broken :+1: 	
in case of acceptance test we are in an odd situation because they eventually should start testing the canary and this field might become forgotten :grimacing:

Reply:
Maybe we should have 2 folders in acceptance tests: 
- renamed `old` to `v2`
- and `canary` for any upcoming changes
Haven't thought about it much yet tho.

Reply:
Yeah acceptance test is tricky, I think we will definitely need the renaming and `canary` folder soon as the current structure is inadequate. Though it also seems to something for larger discussion from its [Readme](https://github.com/TryGhost/Ghost/blob/master/core/test/acceptance/README.md) that we need to figure out its utility structure and reduce some of the tests before start copying them over for `canary` / `v3` 🤔

Reply:
I'd say remove these fields from acceptance tests, so we don't have accidental leftovers. And we'd need to figure out better structure for tests once done with main goal prep.

Reply:
@gargol 👍 Updated now.